gitnamehere/point-of-sale-group-project
# Landing page

**Ticket:** [WEB- Create the landing page for the store site](https://cs375-pos-group-project.atlassian.net/jira/core/projects/ATS/board?selectedIssue=ATS-30)

**PR Desc:**
- Created a landing page for our website
- Implemented Bootstrap to stylize
- Added an about and contact section
- Added a button that will allow the user to enter the website and see the items being sold (not yet implemented/another ticket)

**Testing:**
- Run `npm i`
- Run `npm start`
- Visit http://localhost:3000/store/
- Make sure the navbar works
- Clicking the brand should also bring you back to the top
